src/ansys/turbogrid/core/launcher/launcher.py

```diff
@@ -163,7 +166,40 @@ def launch_turbogrid(
         socket_port=port,
         turbogrid_location_type=pyturbogrid_core.PyTurboGrid.TurboGridLocationType.TURBOGRID_INSTALL,
         cfxtg_location=pathToCFXTG,
-        log_level=log_level,
         additional_args_str=additional_args_str,
         additional_kw_args=additional_kw_args,
+        log_level=log_level,
+        host_ip=host,
+    )
+
+
+def launch_turbogrid_ansys_labs(
+    product_version: str = "latest",
+    log_level: pyturbogrid_core.PyTurboGrid.TurboGridLogLevel = pyturbogrid_core.PyTurboGrid.TurboGridLogLevel.INFO,
+) -> pyturbogrid_core.PyTurboGrid:
+    """Launch TurboGrid locally in server mode.
+
+    Parameters
+    ----------
+    product_version : str, default: ``latest``
+        Version of TurboGrid to launch in the ansys labs environment. Only certain versions are supported.
+    log_level : pyturbogrid_core.PyTurboGrid.TurboGridLogLevel, default: ``INFO``
+        Level of logging information written to the terminal. The default is ``INFO``.
+        Options are ``INFO``, ``WARNING``, ``ERROR``, ``CRITICAL``, and ``DEBUG``.
+        This setting does not affect the level of output that is written to the log files.
```

Comment:
The comment "This setting does not affect the level of output that is written to the log files." is not correct - it was changed some time ago so that the log level did affect the log files. Please could you remove it here, and also at line 141?